lib/nexmo_developer/app/controllers/static_controller.rb

```diff
@@ -45,6 +45,7 @@ def default_landing
       migrate_tropo
     end
 
+    @navigation = request.path.split('/').last.to_sym
```

Comment:
you can use `@navigation = yaml_name` here instead

Reply:
👍